ServerCore/Areas/Identity/Pages/Account/Login.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@page
```

Comment:
Do we actually need this page or can/should we forward directly to the login provider?

Reply:
We go through this page to do the redirect.